### PR TITLE
Transpile to ES5 before publishing npm module by calling build script.

### PR DESCRIPTION
**Pull request by BrerBear**
Closes datavis-tech/json-templates#24

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "name": "json-templates",
-  "version": "2.2.0",
+  "version": "2.3.0",
```
Comment (curran) [inline package.json]: I will undo this and let the NPM tool do this as part of the release process.

File: rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import buble from 'rollup-plugin-buble';
+
+export default {
+  input: 'index.js',
+  output: {
+    format: 'umd',
+    name: 'JSON templates',
```
Comment (curran) [inline rollup.config.js]: The name is used as a global variable name, so I'll change this to something without spaces.

**Comment by curran** [comment]
@BrerBear Released as https://github.com/datavis-tech/json-templates/releases/tag/v2.3.0

Can you please verify that it works in your desired build environment? Thanks!

**Comment by BrerBear** [comment]
react-scripts build system works fine with this update. Thank you!

**Comment by curran** [comment]
Excellent! Thanks for your contribution.

**Pull request merged**